website/cue/reference/remap/functions/for_each.cue

```diff
@@ -0,0 +1,61 @@
+package metadata
+
+remap: functions: for_each: {
+	category: "Enumerate"
+	description: #"""
+		Iterate over a collection.
+
+		This function currently *does not* support recursive iteration.
+		If you have a need for recursive iteration using `for_each`,
+		which can't be solved using the `map_keys` or `map_values`
+		enumeration functions (which *do* support recursion), then
+		please let us know!
```

Comment:
Maybe add a link to a github issue for people to leave comments on?

Reply:
Resolved in [25852d9](https://github.com/vectordotdev/vector/pull/12721/commits/25852d9590d1e062eb7327c4dc3b3fbea6797b29).